.github/workflows/check.yaml

```
@@ -44,7 +44,7 @@ jobs:
       deps-installation-method: setup-r-dependencies
   r-cmd-non-cran:
     name: R CMD Check (non-CRAN) 🧬
-    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
+    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@38b63dee0b54d8b50dd360b1b40258feb081ed54
```

```suggestion
    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
```

Just adding a suggestion to block accidental merge with this.